brandon-brandonmcastillo/week-3/giffaw/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    <meta charset="utf-8">
+		<!-- set viewport to device width to allow responsiveness -->
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<!-- Bootstrap CDN -->
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+
+		<!-- Custom CSS -->
+		<link rel="stylesheet" href="styles/styles.css">
+
+    <!-- JS -->
+		<script src="http://code.jquery.com/jquery-3.3.1.min.js"></script>
+    <script src="scripts/app.js"></script>
+
+		<title>Giffaw</title>
+
+	</head>
+	<body>
+		<div class="container">
+			<h1 class="text-center">Gif Me Up</h1>
+			<div class="row">
+				<div class="col-sm-4">
+					<form class="form-inline">
+						<label>Search Gifs</label>
+						<input type="text" class="form-control gif-input" name="q" value="" id="gif" placeholder="search gifs">
+						<input type="hidden" name="api_key" value="dc6zaTOxFJmzC">
```

Comment:
There was an api key given to you in the html. This is why it's important to read through code is given to you. This will become a very important skill set in the job world as you will always be given code to start with. Make sure you read through all of it to prevent creating more work for yourself than needed.